tasks/python-reference-activity/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@cumulus/python-reference-activity",
```

Comment:
Why does our Python activity task need a package.json file?

Reply:
To run npm scripts/bootstraps/etc as part of the build process (lint, etc)